lib/class-wp-block.php

```diff
@@ -169,19 +169,23 @@ public function __get( $name ) {
 	/**
 	 * Generates the render output for the block.
 	 *
+	 * @param bool $no_dynamic Optionally set to true to avoid using the block's render_callback.
+	 *
 	 * @return string Rendered block output.
 	 */
-	public function render() {
+	public function render( $no_dynamic = false ) {
```

Comment:
We can't use `serialize_block`, because we still want children that are dynamic to render dynamically.

We can't use filters or block type settings because we don't want the change to be permanent, we just want it for some instances.

I've updated it into an options array and flipped the boolean.

We could do it ad-hoc for now, but it pains me to duplicate that code in a block implementation. Can we make the options array experimental instead?